tox.ini

```diff
@@ -26,6 +26,7 @@ deps =
     coverage
     semsimian
     gilda
+    linkml
```

Comment:
a bit worried about duplication of dependency metadata but I don't quite understand why these are in tox.ini in the first place

Reply:
This is because if we run `tox -e py` (to run pytest via tox), it throws errors about linkml module not found (in tot's virtual environment). It doesn't affect `poetry` and it's working in general. We could do away with `tox` and be solely `poetry` dependent but we've had those conversations in the past.